src/sinks/elasticsearch/common.rs

```diff
@@ -118,9 +118,10 @@ impl ElasticsearchCommon {
 
         let metric_config = config.metrics.clone().unwrap_or_default();
         let metric_to_log = MetricToLog::new(
-            metric_config.host_tag,
+            metric_config.host_tag.as_deref(),
             metric_config.timezone.unwrap_or_default(),
             LogNamespace::Legacy,
+            false, // FIXME: Is this right?
```

Comment:
Not quite. The array type could be either a string or `null`, so `null|string|[null|string]` (with a maximum of one `null` in the array). Does this count as being the same type for all elements in the array?

Reply:
> An array may contain null values, which are either replaced by the configured [null_value](https://www.elastic.co/guide/en/elasticsearch/reference/current/null-value.html) or skipped entirely. An empty array [] is treated as a missing field — a field with no values.

So it seems like we can allow users to opt-in/out here. Which may be a boon, because it looks we're hardcoding it to `false` but still rendering the docs for it, but maybe I don't want to wrestle with the docs 😄

Reply:
Fixed in fc50f5bb4